euro-truck-simulator-2config.json

```diff
@@ -0,0 +1,340 @@
+[
+    {
+        "DisplayName":"Lobby Name",
+        "Category":"ETS - Server Settings",
+        "Description":"Session name, limited to 63",
```

Comment:
Missing "characters" at the end.

Reply:
Went ahead and fixed this in ATS on this PR as well. Hope you don't mind.